src/components/overviewPanel/package.json

```diff
@@ -13,8 +13,8 @@
     },
     "dependencies": {
         "jquery": "3.6.0",
-        "opensans-webkit": "1.1.0",
-        "roboto-fontface": "0.10.0",
+        "opensans": "5.0.29",
```

Comment:
@jobara It looks like you have changed to the `@fontsource` namespaced packages elsewhere— is there a reason these haven't changed? I may be misunderstanding the structure or the role of this package.json file.

Reply:
@greatislander these sub package.json files are used for the [internal build system](https://github.com/fluid-project/infusion?tab=readme-ov-file#how-do-i-create-an-infusion-package). They refer to other package listings for internal modules. In this case they are defined in src/lib/package.json which you can see edited in this PR as well.